Tests/iaas/secure-connections/requirements.txt

```diff
@@ -0,0 +1,2 @@
+openstacksdk>=3.0.0
+sslyze>=6.0.0
```

Comment:
TODO: we need proper pinning for this, preferably originating from [Tests/scs-compatible-iaas.yaml](https://github.com/SovereignCloudStack/standards/blob/main/Tests/scs-compatible-iaas.yaml) as discussed with @mbuechse

We need to document the corresponding Mozilla JSON[^1] tightly coupled with the latest SSLyze library version that ships that exact JSON[^2] for testing.


[^1]: https://ssl-config.mozilla.org/guidelines/5.7.json

[^2]: https://github.com/nabla-c0d3/sslyze/blob/761892bf6a613726f272eb4e19b33faee2272b6f/sslyze/mozilla_tls_profile/5.7.json

Requires: #595

Reply:
So, what is the plan going forward? I would assume once https://github.com/SovereignCloudStack/standards/pull/595 is merged, the pinning is done in this PR (548)?

So it seems we need to wait for that? Can't we already prepare some of the necessary steps here?

Reply:
I rewrote parts of the test script now and added a parameterized entry in `Tests/testing/scs-compatible-test.yaml` based on the changes of #595 as an example.